db/fixtures/miq_policy_sets.yml

```diff
@@ -39,6 +39,21 @@
           description: Mark as Non-Compliant
           action_type: default
           options: {}
+      - qualifier: failure
+        failure_sequence: 2
+        failure_synchronous: true
+        MiqEventDefinition:
+          name: containerimage_compliance_check
```

Comment:
@jrafanie @gmcculloug I've create #8806 to track this, feedback on it is appriciated.